nilearn/_utils/estimator_checks.py

```diff
@@ -549,6 +560,27 @@ def check_masker_transformer(estimator):
     assert_array_equal(signal_1, signal_2)
 
 
+def check_masker_transformer_high_variance_confounds(estimator):
+    """Check high_variance_confounds use in maskers.
+
+    Make sure that using high_variance_confounds returns different result.
+    """
+    estimator.high_variance_confounds = False
+
+    if accept_niimg_input(estimator):
+        input_img = _img_4d_rand_eye_medium()
+    else:
+        input_img = _make_surface_img(100)
+
+    signal_1 = estimator.fit_transform(input_img)
+
+    estimator = clone(estimator)
+    estimator.high_variance_confounds = True
+    signal_2 = estimator.fit_transform(input_img)
+
+    assert_raises(AssertionError, assert_array_equal, signal_1, signal_2)
```

Comment:
this is failing for the multi nifti maskers (all of them)

any reason why the multi maskers should not support high_variance_confounds?

Reply:
@bthirion @htwangtw

Reply:
OK will file this as a bug and tackle in a follow up PR